### PR TITLE
obis-observer: add commands

### PR DESCRIPTION
**Pull request by DeadByElpy**

### CODE REVIEW

File: src/obis-observer/message/uplink.ts [hunk 1]
```diff
@@ -95,6 +95,15 @@ toBytesMap[commands.getMeterProfileIdList.id] = commands.getMeterProfileIdList.t
...
+toBytesMap[commands.readMeterArchiveWithDate.id] = commands.readMeterArchiveWithDate.toBytes;
+toBytesMap[commands.reboot.id] = commands.reboot.toBytes;
+toBytesMap[commands.removeMeterProfile.id] = commands.removeMeterProfile.toBytes;
+toBytesMap[commands.removeMeter.id] = commands.removeMeter.toBytes;
+toBytesMap[commands.removeObis.id] = commands.removeObis.toBytes;
+toBytesMap[commands.resetSettings.id] = commands.resetSettings.toBytes;
+toBytesMap[commands.setLorawanActivationMethod.id] = commands.setLorawanActivationMethod.toBytes;
 toBytesMap[commands.setObserverSingleMode.id] = commands.setObserverSingleMode.toBytes;
 toBytesMap[commands.setSerialPort.id] = commands.setSerialPort.toBytes;
 toBytesMap[commands.setupMeterProfile.id] = commands.setupMeterProfile.toBytes;
```
Comment (DarkPark) [inline src/obis-observer/message/uplink.ts:109]: please sort this and other such blocks

File: src/obis-observer/commands/downlink/index.ts [hunk 2]
```diff
@@ -16,6 +16,15 @@ export * as getMeterProfileIdList from './getMeterProfileIdList.js';
 export * as getMeterReadoutState from './getMeterReadoutState.js';
 export * as getObisContent from './getObisContent.js';
 export * as getObisContentById from './getObisContentById.js';
+export * as readArchive from './readArchive.js';
+export * as readMeterArchive from './readMeterArchive.js';
+export * as readMeterArchiveWithDate from './readMeterArchiveWithDate.js';
+export * as reboot from './reboot.js';
+export * as removeMeterProfile from './removeMeterProfile.js';
```
Comment (DarkPark) [inline src/obis-observer/commands/downlink/index.ts:23]: here would be nice to sort as well

**Review by aleks-and — APPROVED** [review]

File: src/obis-observer/message/uplink.ts (hunk 1 above)
Comment (DeadByElpy) [reply src/obis-observer/message/uplink.ts:109]: done

File: src/obis-observer/commands/downlink/index.ts (hunk 2 above)
Comment (DeadByElpy) [reply src/obis-observer/commands/downlink/index.ts:23]: done

File: src/obis-observer/commands/downlink/removeMeter.ts [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+ * const parameters = {
+ *     requestId: 3,
+ *     meterId: 17
+ * };
+ * const bytes = removeMeter.toBytes(parameters);
+ *
+ * // output command binary in hex representation
+ * console.log(bytes);
+ * // output
+ * [114, 3, 0, 0, 0, 17]
```
Comment (DarkPark) [inline src/obis-observer/commands/downlink/removeMeter.ts:19]: is it correct?

File: src/obis-observer/commands/downlink/removeMeterProfile.ts [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+ * const parameters = {
+ *     requestId: 3,
+ *     meterProfileId: 17
+ * };
+ * const bytes = removeMeterProfile.toBytes(parameters);
+ *
+ * // output command binary in hex representation
+ * console.log(bytes);
+ * // output
+ * [98, 3, 17]
```
Comment (DarkPark) [inline src/obis-observer/commands/downlink/removeMeterProfile.ts:19]: does not match the example below

File: src/obis-observer/commands/downlink/removeMeter.ts (hunk 3 above)
Comment (DeadByElpy) [reply src/obis-observer/commands/downlink/removeMeter.ts:19]: fixed

File: src/obis-observer/commands/downlink/removeMeterProfile.ts (hunk 4 above)
Comment (DeadByElpy) [reply src/obis-observer/commands/downlink/removeMeterProfile.ts:19]: fixed

**Review by DarkPark — APPROVED** [review]

**Pull request merged**